src/mvt/android/modules/androidqf/files.py

```diff
@@ -106,6 +107,12 @@ def check_indicators(self) -> None:
             # TODO: adds SHA1 and MD5 when available in MVT
 
     def run(self) -> None:
+        if timezone := self._get_device_timezone():
+            device_timezone = ZoneInfo(timezone)
+        else:
+            self.log.warning("Unable to determine device timezone, using UTC")
+            device_timezone = ZoneInfo("UTC")
```

Comment:
Will look after lunch but either a `backports` or `pytz` should fix the CI failure here